Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.py

```diff
@@ -407,7 +407,7 @@ def run_queries_command(client: Client, args: Dict, optional_args=None):
     target = args.get('target')
     node = {'id': args.get('id')}
     if target:
-        node['target'] = {'ids': args.get('target')}
+        node['target'] = {'ids': [args.get('target')]}
```

Comment:
assuming this can be a single argument or a list this should probably use argToList
```suggestion
        node['target'] = {'ids': argToList(args.get('target'))}
```